.godocdown.template

```diff
@@ -1,7 +1,21 @@
 {{ .Emit }}
 
+## Development
+Tests expects the files to be in `$GOPATH/src/gopkg.in/Clever/optimus.v3` to work properly
```

Comment:
Can we change the language here to make this more general? Something like "You must develop Go packages from inside your Go path. For `optimus`, that means that you must be in `$GOPATH/src/gopkg.in/Clever/optimus.v3`"